src/components/Contentful/DynamicHeroClassic.vue

```diff
@@ -108,10 +109,27 @@
 						<div v-if="heroBody" class="tw-prose tw-mb-2 md:tw-mb-3" ref="heroBodyCopy">
 							<dynamic-rich-text :html="heroBody" />
 						</div>
-						<button-wrapper
-							class="tw-w-full md:tw-w-auto"
-							:content="buttonContent"
-						/>
+						<div v-if="gridHeroBody" class="tw-mb-2 md:tw-mb-3">
+							<dynamic-rich-text :body-columns="bodyColumns" :html="gridHeroBody" />
```

Comment:
I think the body-columns prop should be applied to the existing heroBody dynamic rich text, rather than having a separate "gridHeroBody".